searchfingerprint

```diff
@@ -40,8 +40,15 @@ while [ "${*}" != "" ] ; do
     shift
 
     #Confirm input is a video file
-    if [[ -z $(file -i "${INPUT}" | grep video) ]] ; then
-        echo "Input is not a video file" && continue
+    if [[ "$(uname -s)" = Darwin ]] ; then
+        VIDEOCHECK='file -Ib'
+    elif [[ "$(uname -s)" = Linux ]] ; then
+        VIDEOCHECK='file -i'
+    fi
```

Comment:
I would add also here an
```
else
```
with a warning or an error.

Reply:
Since it is just a check to see if the input is a video (and throw an error if it isn't) I thought it would be better to attempt to set the variable, but then to skip the video check if the variable was unset (See line 48).

This way, if the person was using an irregular system (but somehow everything else was working) with a valid input it would allow them to proceed.

Reply:
I understand. Would in this case an
```
unset VIDEOCHECK
```
before the line 43 be appropriate?

Reply:
good point! I'll add that in!